specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/examples/cloud-events-schema/chat_azure_bot_command_received_in_thread.json

```diff
@@ -0,0 +1,36 @@
+{
+  "source": "/subscriptions/{subscription-id}/resourceGroups/{group-name}/providers/Microsoft.Communication/communicationServices/{communication-services-resource-name}",
+  "subject": "thread/{thread-id}/sender/{rawId}/recipient/{rawId}",
+  "type": "Microsoft.Communication.ChatAzureBotCommandReceivedInThread",
+  "time": "2021-02-19T00:25:59.9436666Z",
+  "id": "02272459-badb-4e2e-b538-4cb8a2f71da6",
+  "data": {
+    "messageBody": "Welcome to Azure Communication Services",
+    "messageId": "1613694358927",
+    "metadata": {
+      "key": "value",
+      "description": "A map of data associated with the message"
+    },
+    "senderId": "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7caf-07fd-084822001724",
```

Comment:
Is this copy/pasted from our internal code? If yes, it maybe exposing internal resource info. Generate a new Guid and copy here instead.